src/CMakeLists.txt

```diff
@@ -187,6 +185,9 @@ set ( libfluidsynth_SOURCES
     bindings/fluid_ladspa.c
     bindings/fluid_ladspa.h
 )
+
+file ( GLOB fluid_osal_SOURCES utils/fluid_sys_${osal}.* )
```

Comment:
Please refrain from file globbing. This won't work as expected when git bisecting through the code.

Reply:
I have now added a version that selects .cpp or .c depending on osal == cpp11.